firstvoices/backend/models/dictionary.py

```diff
@@ -194,11 +196,19 @@ def set_custom_order(self, alphabet):
 
 
 class DictionaryEntryLink(BaseSiteContentModel):
```

Comment:
This could possibly use a string function here to improve readability, mainly in the admin panel, which returns something like `self.to_dictionary_entry.title`.